source/whats-new.txt

```diff
@@ -33,6 +41,30 @@ Learn what's new in:
 * :ref:`Version 1.1 <version-1.1>`
 * :ref:`Version 1.0 <version-1.0>`
 
+.. _version-1.13:
+
+What's New in 1.13
+------------------
+
+The 1.13 {+driver-short+} release includes the following improvements and fixes:
+
+- Logging for server selection and SDAM. See the :ref:`golang-logging` documentation
+  for more information about logging.
+
+- Methods on the ``mongo.Collection`` type for managing search indexes programmatically.
+
+- The ``event.CommandStartedEvent`` and ``event.CommandFinishedEvent`` events return
+  the ``DatabaseName`` field. This field is also included in the command-logging analogs.
+
+- In a sharded topology, when selecting a server to retry a failing query, the driver
+  excludes the server used for the initial failed attempt. Instead, if there's more than
+  one eligible ``mongos`` instance, the driver randomly selects one. Unhealthy instances
+  are automatically excluded from selection. 
+
+- Streaming SDAM is disabled by default on AWS Lambda and similar function-as-a-service
+  (FaaS) platforms. You can change the monitoring mode by using the
+  ``serverMonitoringMode`` URI option.
```

Comment:
S: to avoid passive voice and vale error, what if you reworded as follows:
```suggestion
- You can enable streaming SDAM events on AWS Lambda and similar function-as-a-service
  (FaaS) platforms by changing the monitoring mode. You can change the monitoring mode by setting
  the ``serverMonitoringMode`` URI option to....
```

Reply:
- seems relevant to mention that it's disabled by default on these platforms? especially if users are assuming it's going to be enabled.
- i intentionally omitted the enabling code from this section, figuring we'd put it elsewhere in the docs. do you think it's particularly helpful here?

Reply:
I dont think that level of detail is necessary for the whats new section.